app/Http/Resources/Group.php

```diff
@@ -279,8 +313,10 @@ public function toArray($request)
             'stats' => $stats,
             'updated_at' => Carbon::parse($this->updated_at)->toIso8601String(),
             'location' => new GroupLocation($this),
-            'networks' => $this->resource->networks,
+            'networks' => new NetworkSummaryCollection($this->networks),
+            'tags' => new TagCollection($this->group_tags),
```

Comment:
I think here we have an issue to think about, in that some of the data we use internally, is not really relevant (and perhaps shouldn't be seen at all) to our current API users.  For example, at present group tags are only set by admins, yet we're going to be returning them to clients of the API.  Not a big issue for now I don't think, but this general distinction of who sees what via the API would be good to discuss.  We kind of have two use cases - the API for *our client*, and the API to return certain data we want partners to see.  Maybe it's fine for those both to end up being the same thing, but we should discuss.

Reply:
As discussed at retro.